cogs/tags.py

```diff
@@ -21,6 +22,35 @@ def build_tag_name(record):
         name += f" ({record.get('alias')})"
     return name
 
+async def convert_slash(self, inter, tag_name):
+    tag_name = tag_name.lower()
+
+    rec = await inter.bot.db.fetchrow(
+        "SELECT * FROM tag WHERE guild_id=$1 AND (name=$2 OR alias=$2)",
+        inter.guild.id,
+        tag_name,
+    )
+
+    if rec is not None:
+        return rec
+
+    # otherwise, find a list of potential matches
+
+    similars = await inter.bot.db.fetch(
+        "SELECT name, alias FROM tag WHERE guild_id=$1 AND (name LIKE '%$2%' OR alias LIKE '%$2%') ORDER BY uses DESC, viewed_at DESC, edited_at DESC, created_at DESC LIMIT 5",
```

Comment:
seems like an excessive amount of ORDER BY values, like the `edited_at DESC` will only *ever* do anything if there are multiple tags with the same `viewed_at` value, which is just exceedingly unlikely, and the same logic continues on for `created_at`